dev-libs/gtest/gtest-1.9.0~git.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="C++ test framework"
+DESCRIPTION="C++ testing utility
+Features:
+* An xUnit test framework
+* Test discovery
+* A rich set of assertions
+* User-defined assertions
+* Death tests
+* Fatal and non-fatal failures
+* Value-parameterized tests
+* Type-parameterized tests
+* Various options for running the tests
+* XML test report generation"
+HOMEPAGE="https://github.com/google/googletest/"
+COPYRIGHT="2008-2016 Google Test developers"
+LICENSE="BSD (3-clause)"
+srcGitRev="ea31cb15f0c2ab9f5f5b18e82311eb522989d747"
+REVISION="1"
+SOURCE_URI="https://github.com/google/googletest/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="41a782bacfed269409e533599f2272910e6c7ccd0c3d03f9694c693916070180"
+SOURCE_FILENAME="googletest-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="googletest-$srcGitRev"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gtest$secondaryArchSuffix = $portVersion
+	lib:libgmock$secondaryArchSuffix = $portVersion
+	lib:libgmock_main$secondaryArchSuffix = $portVersion
+	lib:libgtest$secondaryArchSuffix = $portVersion
+	lib:libgtest_main$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	gtest${secondaryArchSuffix}_devel = $portVersion
+	devel:libgmock$secondaryArchSuffix = $portVersion
+	devel:libgmock_main$secondaryArchSuffix = $portVersion
+	devel:libgtest$secondaryArchSuffix = $portVersion
+	devel:libgtest_main$secondaryArchSuffix = $portVersion
+	"
+REQUIRES_devel="
+	gtest$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	cmd:python3
```

Comment:
Python3 has support for secondary architectures, so I guess it would be safer to use it here.
```diff
-	cmd:python3
+	cmd:python3$secondaryArchSuffix
```

Reply:
@ your service :)

Reply:
Need to revert this, looks like python3_x86 doesn't provide the cmd:python3_x86 :)

Reply:
Oops, my mistake! I'm so sorry!

Well, if, on x86_gcc2, haikuporter activates python3 instead of python3_x86, then it might be worth trying to "convince" it to activate python3_x86.
Maybe adding `devel:libpython3.6m$secondaryArchSuffix` to `BUILD_REQUIRES` will do the trick. Good luck :)

Reply:
Beware, python3_x86 seems to have problems with library paths and may crash.

Reply:
Should we use python2 here instead then? I don't see any errors on x86_gcc2 (x86) nor x8_64